[{"lineNumber": 1, "col_offset": 26, "nodeName": "Exception", "type": "Type[Exception]"}, {"lineNumber": 7, "col_offset": 23, "nodeName": "EvaluationException", "type": "Type[EvaluationException]"}, {"lineNumber": 15, "col_offset": 27, "nodeName": "EvaluationException", "type": "Type[EvaluationException]"}, {"lineNumber": 23, "col_offset": 22, "nodeName": "EvaluationException", "type": "Type[EvaluationException]"}, {"lineNumber": 31, "col_offset": 32, "nodeName": "EvaluationException", "type": "Type[EvaluationException]"}, {"lineNumber": 3, "col_offset": 8, "nodeName": "message", "type": "Any"}, {"lineNumber": 3, "col_offset": 23, "nodeName": "message", "type": "Any"}, {"lineNumber": 3, "col_offset": 8, "nodeName": "self", "type": "EvaluationException"}, {"lineNumber": 4, "col_offset": 8, "nodeName": "__init__", "type": "Callable[..., None]"}, {"lineNumber": 4, "col_offset": 8, "nodeName": "super", "type": "Type[super]"}, {"lineNumber": 4, "col_offset": 14, "nodeName": "EvaluationException", "type": "Type[EvaluationException]"}, {"lineNumber": 4, "col_offset": 35, "nodeName": "self", "type": "EvaluationException"}]